exporter/cat1_view_helper.go

```diff
@@ -253,20 +262,14 @@ func hasPreferredCode(pc models.Concept) bool {
 	return pc.Code != "" && pc.CodeSystem != ""
 }
 
+func hasLaterality(l models.Laterality) bool {
```

Comment:
It isn't gone, he moved it to the proper struct it belongs on. Procedure, I think?